resources/services/backup/vaults.go

```diff
@@ -365,13 +367,37 @@ func resolveVaultRecoveryPointCreatedBy(ctx context.Context, meta schema.ClientM
 
 func resolveRecoveryPointTags(ctx context.Context, meta schema.ClientMeta, resource *schema.Resource, c schema.Column) error {
 	rp := resource.Item.(types.RecoveryPointByBackupVault)
+	if rp.ResourceArn == nil || rp.RecoveryPointArn == nil {
+		return nil
+	}
+	resourceARN, err := arn.Parse(*rp.ResourceArn)
+	if err != nil {
+		return diag.WrapError(err)
+	}
+
+	// decide if the backed up resource supports tags
+	switch client.AWSService(resourceARN.Service) {
+	case client.S3Service, client.EFSService, client.DynamoDBService:
```

Comment:
empty case here to allow tags might raise eyebrows on reviews, maybe a comment?

Reply:
:shrug: added a comment